onnxruntime/core/providers/webgpu/buffer_manager.h

```diff
@@ -14,24 +14,34 @@ namespace webgpu {
 
 class WebGpuContext;
 
+// For command capture and replay
+enum class SStatus {
```

Comment:
"SStatus" looks strange. maybe `SessionCapturingState` with `NotStarted`, `Capturing`, `Captured`?

Reply:
Rename it to `SessionStatus` and keep the original enum value. The `session_status_` in webgpu_context is the current active session's status. Only if the current session is capturing or replaying, the `session_status_` is `Capturing` or `Replaying`. Otherwise, `session_status_` should be always in `Default` mode.